api/src/main/java/io/opencensus/trace/SpanId.java

```diff
@@ -83,6 +83,20 @@ public static SpanId fromBytes(byte[] src, int srcOffset) {
     return new SpanId(bytes);
   }
 
+  /**
+   * Returns a {@code SpanId} built from a lowercase base16 representation.
+   *
+   * @param src the lowercase base16 representation.
+   * @return a {@code SpanId} built from a lowercase base16 representation.
+   * @throws NullPointerException if {@code src} is null.
```

Comment:
Maybe add `throws IllegalArgumentException` in javadoc as `TraceId.fromLowerBase16` does?